services/notify/src/main.js

```diff
@@ -136,9 +125,9 @@ const load = loader({
   },
 
   denier: {
-    requires: ['cfg', 'DenylistedNotification'],
-    setup: ({cfg, DenylistedNotification}) =>
-      new Denier({DenylistedNotification, emailBlacklist: cfg.app.emailBlacklist}),
+    requires: ['cfg', 'db'],
+    setup: ({cfg, db}) =>
+      new Denier({emailBlacklist: cfg.app.emailBlacklist, db: db}),
```

Comment:
This could have been written more briefly with `new Denier({emailBlacklist: cfg.app.emailBlacklist, db}),`. There's nothing wrong with ^ however so no need to make any changes.